lib/utils.js

```diff
@@ -25,54 +30,77 @@ function toConventionalChangelogFormat (ast) {
       case 'summary':
         summary = node
         break
-      default:
-        break
     }
   })
 
+  // <type>, "(", <scope>, ")", ["!"], ":", <whitespace>*, <text>
   visit(summary, () => true, (node) => {
```

Comment:
You don't have to provide `() => true`, you can just write it as `visit(summary, (node) => ...`